src/PhpSpreadsheet/Collection/Cells.php

```diff
@@ -209,7 +209,7 @@ public function getCurrentColumn()
         $column = '';
         $row = 0;
 
-        sscanf($this->currentCoordinate, '%[A-Z]%d', $column, $row);
+        sscanf($this->currentCoordinate ?? 'A1', '%[A-Z]%d', $column, $row);
```

Comment:
Then shouldn't we do something like:

```php
/**
 * Return the column coordinate of the currently active cell object.
 *
 * @return null|string
 */
public function getCurrentColumn()
{
    $column = null;
    $row = null;

    if ($this->currentCoordinate) {
        sscanf($this->currentCoordinate, '%[A-Z]%d', $column, $row);
    }

    return $column;
}
```

Reply:
Done.

Reply:
Changing the function to return null|string starts a game of whack-a-mole as callers now appear to have unexpected results. I have coerced to null-string instead of A1 - caller will not get a misleading result, and can test for this condition.